static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -242,11 +252,11 @@ function GroupReplaysTable({
   return (
     <StyledLayoutPage withPadding>
       <ReplayCountHeader>
-        <StyledIconUser size="sm" />
+        <IconUser size="sm" />
         {t(
-          'Replay captured %s users experiencing this issue across %s events.',
-          replayCount,
-          group.count
+          'Replay captured %s experiencing this issue across %s.',
+          tn('%s user', '%s users', replayCount ?? 0),
+          tn('%s event', '%s events', group.count)
```

Comment:
Could we put the whole sentence in to the ternary and tn() calls. drop the t() wrapper part. Should be easier for  translators to have the full sentence there.

Reply:
i was thinking you could have 1 user but 2 events or 2 events and 1 user

Reply:
Fair enough. 

Tough to use t18n lib :(